homeassistant/components/camera/amcrest.py

```diff
@@ -6,23 +6,31 @@
 """
 import logging
 
+import asyncio
+import aiohttp
+from aiohttp import web
+from aiohttp.web_exceptions import HTTPGatewayTimeout
+import async_timeout
 import voluptuous as vol
 
 import homeassistant.loader as loader
 from homeassistant.components.camera import (Camera, PLATFORM_SCHEMA)
 from homeassistant.const import (
     CONF_HOST, CONF_NAME, CONF_USERNAME, CONF_PASSWORD, CONF_PORT)
 from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
 
 REQUIREMENTS = ['amcrest==1.0.0']
```

Comment:
PR #5310 bumps the version to 1.1.0.  I don't strictly need the newer version here, so I'll leave this as-is (and we'll let #5310 bump the version when that's merged in).